docker/grpc-exporter/app/server.py

```diff
@@ -0,0 +1,69 @@
+import os
+import time
+import subprocess
+import json
+from prometheus_client import start_http_server, Gauge
+
+
+class AppMetrics:
+    """
+    Representation of Prometheus metrics and loop to fetch and transform
+    application metrics into Prometheus metrics.
+    """
+
+    def __init__(self, polling_interval_seconds=60):
+        self.polling_interval_seconds = polling_interval_seconds
+
+        # Prometheus metrics to collect
+        self.ourMessages = Gauge("our_message_height", "Our Messages", ['peerId'])
```

Comment:
I worry that with permissionless hubs, the number of peers is going to grow greatly and may bloat our prometheus datastore